biom/cli/uc_processor.py

```diff
@@ -0,0 +1,86 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2011-2013, The BIOM Format Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from __future__ import division
+
+import click
+
+from biom.cli import cli
+from biom.cli.util import write_biom_table
+from biom.parse import parse_uc
+from biom.exception import TableException
+
+
+@cli.command('from-uc')
+@click.option('-i', '--input-fp', required=True,
+              type=click.Path(exists=True, dir_okay=False),
+              help='The input uc filepath.')
+@click.option('-o', '--output-fp', default=None,
+              type=click.Path(writable=True),
+              help='The output BIOM filepath', required=False)
+@click.option('--rep-set-fp', type=click.Path(exists=True, dir_okay=False),
+              help="Fasta file containing representative sequences with "
+                   "where sequences are labeled with OTU identifiers, and "
+                   "description fields contain original sequence identifiers. "
+                   "This output is created, for example, by vsearch with the "
+                   "--relabel_sha1 --relabel_keep options.",
+              required=False)
+def from_uc(input_fp, output_fp, rep_set_fp):
+    """Create a BIOM table from a vsearch/uclust/usearch BIOM file.
+
+    Example usage:
+
+    Simple BIOM creation:
+
+    $ biom from-uc -i in.uc -o out.biom
+
+    BIOM creation with OTU re-naming:
+
+    $ biom from-uc -i in.uc -o out.biom --rep-set-fp rep-set.fna
+
+    """
+    input_f = open(input_fp, 'U')
+    if rep_set_fp is not None:
+        rep_set_f = open(rep_set_fp, 'U')
+    else:
+        rep_set_f = None
+    table = _from_uc(input_f, rep_set_f)
+    write_biom_table(table, 'hdf5', output_fp)
```

Comment:
Yep to both (see ``write_biom_table``), can always convert. I would like to use the same writing approach as the other scripts.